lib/data/network/contact/lookup_api.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:fluffychat/data/network/dio_client.dart';
+import 'package:fluffychat/data/network/identity_endpoint.dart';
+import 'package:fluffychat/di/global/get_it_initializer.dart';
+import 'package:fluffychat/di/global/network_di.dart';
+import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
+import 'package:fluffychat/domain/model/contact/lookup_list_mxid_request.dart';
+import 'package:fluffychat/domain/model/contact/lookup_list_mxid_response.dart';
+
+class LookupAPI {
+  final DioClient _client =
+      getIt.get<DioClient>(instanceName: NetworkDI.tomDioClientName);
```

Comment:
IMO, lookup should use `identity` dio instance

Reply:
No, it is lookup matrixId on Tom Contacts, show it is from Tom
```
https://tom.linagora.com/_matrix/identity/v2/hash_details
```